### PR TITLE
Added List of Student Campus Ambassador Programs

### PR DESCRIPTION
**Pull request by 0xquark**
This list will help students to get the opportunity to learn outside the campus

### CODE REVIEW

**Comment by closebracket** [comment]
Hi @0xquark,

Thanks for submitting to the Freeside Student Resources List.

Firstly, Could you confirm that you're submitting as part of Hacktoberfest? There doesn't seem to be any notes relating to this so I can assign the label if that's the case.

Secondly, You've submitted some very helpful resources. I think students at Hull (and perhaps staff too) will find the Ambassador programmes you have listed beneficial. Unfortunately, not all of the links work and I'll highlight those in the code. If you could correct that, perhaps by adding in other programmes, that would be great.

Finally, the contents link for the category isn't quite correct. Could I ask that you correct this as well, please?

If you can correct those things, I'm happy to accept the PR.

Many thanks,

CB

File: README.md [hunk 1]
```diff
@@ -163,6 +164,25 @@ If you require any support, either whilst contributing or with linux, programmin
 6. [OpenSSH Manual](https://www.openssh.com/manual.html)
 7. [Principle of Least Privilege](https://techcommunity.microsoft.com/t5/azure-sql/security-the-principle-of-least-privilege-polp/ba-p/2067390)
 
+## Campus Ambassador Programs <a name="Campus Ambassador Programs"></a>
+1. [Microsoft Learn Student Ambassadors](https://studentambassadors.microsoft.com/) 
+2. [GitHub Campus Experts](https://githubcampus.expert/)
+3. [College Representative  - E-Cell IITB](https://www.ecell.in/cr/)
+4. [Internshala Student Partner  -   | ISP](https://internshala.com/)
+5. [Progate Student Ambassador](http://progate.com/)
+6. [ISB - YLP Campus Ambassador Program ](http://www.isb.edu/ylp/CAP)
```
Comment (closebracket) [inline README.md:173]: This link redirects to 404

File: README.md [hunk 2]
```diff
@@ -163,6 +164,25 @@ If you require any support, either whilst contributing or with linux, programmin
...
+7. [GeeksforGeeks Campus Ambassador](https://www.geeksforgeeks.org/)
+8. [HackerEarth Campus Ambassador](https://hackerearth.com)
```
Comment (closebracket) [inline README.md:175]: This doesn't appear to be a link to the ambassador programme. Can you provide a link directly to the programme?

File: README.md [hunk 3]
```diff
@@ -163,6 +164,25 @@ If you require any support, either whilst contributing or with linux, programmin
...
+9. [HackerRank Campus Ambassador](https://hackerrank.com)
```
Comment (closebracket) [inline README.md:176]: This doesn't appear to be a link to the ambassador programme. Can you provide a link directly to the programme?

File: README.md [hunk 4]
```diff
@@ -163,6 +164,25 @@ If you require any support, either whilst contributing or with linux, programmin
...
+10. [Interviewbit Campus Ambassador](https://www.interviewbit.com/pages/campus-ambassador/)
+11. [Dell Campassadors Program](https://dellfuturist.com/the-dell-campassadors-program)
```
Comment (closebracket) [inline README.md:178]: This redirects to 404

File: README.md [hunk 5]
```diff
@@ -163,6 +164,25 @@ If you require any support, either whilst contributing or with linux, programmin
...
+12. [Intel Ambassador Program](https://software.intel.com/en-us/ai-academy/ambassadors/apply)
+13. [Codechef Campus Ambassador](https://www.codechef.com/)
+14. [Ingressive Campus Ambassador](https://www.ingressive.co/ingressive-campus-ambassadors/)
```
Comment (closebracket) [inline README.md:181]: Redirects to 404

File: README.md [hunk 6]
```diff
@@ -163,6 +164,25 @@ If you require any support, either whilst contributing or with linux, programmin
...
+15. [Mozilla Student Ambassador](https://www.mozilla.org/en-US/contribute/studentambassadors/)
+16. [Frontbench Campus Ambassador](https://frontbench.xyz/campus-connect)
```
Comment (closebracket) [inline README.md:183]: Site offline

File: README.md [hunk 7]
```diff
@@ -163,6 +164,25 @@ If you require any support, either whilst contributing or with linux, programmin
...
+17. [Codechef Campus Ambassador](https://www.codechef.com/college-chapter/about)
```
Comment (closebracket) [inline README.md:184]: Duplicate site - please remove the other codechef link.

**Comment by 0xquark** [comment]
Thank you for reviewing my PR . I am submitting on behalf of Hactoberfest so please do assign me with that label. I will make necessary changes on your recommendations . :) 
Also I am new to open source , sincere Apologies for my mistakes.

Edit : I just checked some of the sites are not working , I will replace them with working links. Also I will add some more campus ambassador program sites to the repo. Thank you again for taking your time to review these and add comments

**Comment by closebracket** [comment]
Closing this PR as it hasn't been updated in 3 days.

**Comment by closebracket** [comment]
Reopening after reconsidering the length PRs should be given before closing. @0xquark please feel free to continue updating and submit your PR.

**Comment by closebracket** [comment]
@0xquark Are you still working on this?

**Comment by closebracket** [comment]
Hacktoberfest contribution past the deadline. Closing.

**Comment by sbrl** [comment]
> Hacktoberfest contribution past the deadline. Closing.

So what? By using the `hacktoberfest-accepted` tag it can still count. Besides, it's better to judge PRs based on their actual content and hacktoberfest second.

**Comment by closebracket** [comment]
> So what? By using the `hacktoberfest-accepted` tag it can still count. Besides, it's better to judge PRs based on their actual content and hacktoberfest second.

1. I asked 22 days ago if the contributor was still working on it. There was no response. You had previously said that I should wait longer for people to respond to requests for changes/comments etc. I did. There was still no response.
2. Whilst it is true that "_Pull requests that are still in review after Oct 31 and meet the criteria will count towards your completion goal." (source: Hacktoberfest rules), the PRs I closed have been inactive since Oct 4. That is over a month. 
3. As can be seen from the above comments, there were also issues with the original submission quality; so whilst I agree that submissions should be judged on quality as well, this PR failed on that count as well. 

On that basis, the PR was closed and I feel justified in my decision. You are of course, welcome to override my decision if others feel the same way as yourself.

**Comment by sbrl** [comment]
Very true, @closebracket. Not saying your decision was unnjustified - far from it - rather that your explanation was unclear to me :-)

**Comment by closebracket** [comment]
I will endeavour to give better reasoning in future my rainbow loving friend 👍🏻

**Pull request closed without merging**